app/views/users/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<% if @user.errors.any? %> <div id="error_explanation"> <h2><%= pluralize(@user.errors.count, "error") %> prohibited this post from being saved:</h2> <ul> <% @user.errors.full_messages.each do |msg| %> <li><%= msg %></li> <% end %> </ul> </div> <% end %> 
+<%= form_for @user do |f|%>
+
+<label for="username">Username</label>
+<%= f.text_field :username%>
+<br>
+<label for="email">Email</label>
+<%= f.email_field :email%>
+<br>
+<label for="password">Password</label>
+<%= f.password_field :password%>
+<br>
+<%= f.submit "Submit"%>
+<%end%>
+
+<%# the text below are the examples of the old versions of forms we did before the updated one above, we kept it as advised by TheOdinProject%>
+
+
+<% if false%>
+<%= form_tag(users_path, method: "POST") do %>
+     <%= label_tag(:username, "Username:") %>
+     <%= text_field_tag(:username) %><br><br>
+     <%= label_tag(:email, "email:") %>
+     <%= email_field_tag(:email) %><br><br>
+     <%= label_tag(:password, "password:") %>
+     <%= password_field_tag(:password) %><br><br>
+     <%= submit_tag("Submit")%>
+<% end %>
+<%end%>
+
+
+<% if false %>
+<form accept-charset="UTF-8" action="<%= users_path %>" method="POST">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <input type="submit" value="Submit">
+</form>
+<% end %>
```

Comment:
This form should also be commented out 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

Use the links as a reference.

Reply:
they are commented, using 
<% if false %>
     code here 
<% end %>
this is the best way i found to comment in an erb file as i saw in many posts on stack overflow.